lib/msf/core/handler/reverse.rb

```diff
@@ -82,12 +93,16 @@ def setup_handler
                 'MsfPayload' => self,
                 'MsfExploit' => assoc_exploit
               })
+
+            if is_loopback_address(ip)
```

Comment:
What specifically needs to happen here? I've modified the indentation so it's not inline with `begin` -- is the issue the whitespace on 96? I'm using Atom to edit the file.